### PR TITLE
Create Thats crazy

### PR DESCRIPTION
**Pull request by SME684**

### CODE REVIEW

**Review by SME684 — COMMENTED** [review]
Nice

**Review by zachattack234 — APPROVED** [review]
lets find the next step

File: Thats crazy [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This is pretty cool
+Like for real
```
Comment (zachattack234) [inline Thats crazy:2]: its basically like coding huh?

**Review by zachattack234 — CHANGES_REQUESTED** [review]

File: Thats crazy (hunk 1 above)
Comment (SME684) [reply Thats crazy:2]: ya very cool

**Pull request merged**